### PR TITLE
Make compatible with pac4j 6.0.0-RC7

### PR DESCRIPTION
**Pull request by spinscale**
This removes the JavalinWebContext and uses only servlet calls.

Note: This is WIP! Currently the `JavalinPac4jExample` class allows every call endpoint, if logged in once, and I do not know whether this is the correct approach or not - checked one previous version and this behaviour already existed, so it's ok?

### CODE REVIEW

File: src/main/java/org/pac4j/javalin/CallbackHandler.java [hunk 1]
```diff
@@ -36,19 +32,20 @@ public CallbackHandler(Config config, String defaultUrl, Boolean renewSession) {
...
     public void handle(@NotNull Context javalinCtx) {
-        final SessionStoreFactory sessionStoreFactory = FindBest.sessionStoreFactory(null, config, JEESessionStoreFactory.INSTANCE);
-        final SessionStore sessionStore = sessionStoreFactory.newSessionStore(javalinCtx);
-        final HttpActionAdapter bestAdapter = FindBest.httpActionAdapter(null, config, JavalinHttpActionAdapter.INSTANCE);
-        final CallbackLogic bestCallbackLogic = FindBest.callbackLogic(null, config, DefaultCallbackLogic.INSTANCE);
-
-        JavalinWebContext context = new JavalinWebContext(javalinCtx);
-        bestCallbackLogic.perform(context,
-                sessionStore,
+        final CallbackLogic callbackLogic;
```
Comment (leleuj) [inline src/main/java/org/pac4j/javalin/CallbackHandler.java:35]: No. You should use: https://github.com/pac4j/jee-pac4j/blob/master/jakartaee-pac4j/src/main/java/org/pac4j/jee/filter/CallbackFilter.java#L61

File: src/main/java/org/pac4j/javalin/LogoutHandler.java [hunk 2]
```diff
@@ -39,21 +35,21 @@ public LogoutHandler(Config config, String defaultUrl, String logoutUrlPattern)
...
     public void handle(@NotNull Context javalinCtx) {
-        final SessionStoreFactory sessionStoreFactory = FindBest.sessionStoreFactory(null, config, JEESessionStoreFactory.INSTANCE);
-        final SessionStore sessionStore = sessionStoreFactory.newSessionStore(javalinCtx);
-        final HttpActionAdapter bestAdapter = FindBest.httpActionAdapter(null, config, JavalinHttpActionAdapter.INSTANCE);
-        final LogoutLogic bestLogic = FindBest.logoutLogic(null, config, DefaultLogoutLogic.INSTANCE);
-
-        bestLogic.perform(
-            new JavalinWebContext(javalinCtx),
-            sessionStore,
+        final LogoutLogic logoutLogic;
```
Comment (leleuj) [inline src/main/java/org/pac4j/javalin/LogoutHandler.java:38]: Same here

File: src/main/java/org/pac4j/javalin/SecurityHandler.java [hunk 3]
```diff
@@ -41,21 +37,20 @@ public SecurityHandler(Config config, String clients, String authorizers, String
 
     @Override
     public void handle(@NotNull Context javalinCtx) {
-        final SessionStoreFactory sessionStoreFactory = FindBest.sessionStoreFactory(null, config, JEESessionStoreFactory.INSTANCE);
-        final SessionStore sessionStore = sessionStoreFactory.newSessionStore(javalinCtx);
-        final HttpActionAdapter bestAdapter = FindBest.httpActionAdapter(null, config, JavalinHttpActionAdapter.INSTANCE);
-        final SecurityLogic bestLogic = FindBest.securityLogic(null, config, DefaultSecurityLogic.INSTANCE);
+        final SecurityLogic securityLogic;
```
Comment (leleuj) [inline src/main/java/org/pac4j/javalin/SecurityHandler.java:40]: And again

**Review by leleuj — CHANGES_REQUESTED** [review]

**Comment by leleuj** [comment]
As soon as you are authenticated, you can access any protected endpoint (if no specific authorization is requested), the `client` only defines the authentication method when you are not authenticated.

File: src/main/java/org/pac4j/javalin/CallbackHandler.java (hunk 1 above)
Comment (spinscale) [reply src/main/java/org/pac4j/javalin/CallbackHandler.java]: aaah... done

**Pull request merged**